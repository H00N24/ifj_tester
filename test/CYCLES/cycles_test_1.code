// Rozsirenie CYCLES

class Main
{
	static void run()
	{
		int x = 5;
		int k = 0;

		for (int i = 0; i < 5; i = i + 1)
		{
			ifj16.print(k + ". Iteracia" + "\n");

			if (k == 0)
			{
				do
				{
					ifj16.print("Aspon raz" + "\n");
				} while (x != 5);

				x = x - 5;

				do
				{
					x = x + 1;
					ifj16.print("Pred continue" + "\n");
					continue;

					ifj16.print("Tu nemam byt" + "\n");
				} while (x < 5);
			}

			ifj16.print("Koniec?" + "\n");

			if (x == 25)
				break;
			else
			{
				x = 25;
				k = 1;
				continue;

				ifj16.print("Tu nemam byt" + "\n");
			}
		}
	}
}