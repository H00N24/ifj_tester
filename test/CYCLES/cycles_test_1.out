0. Iteracia
Aspon raz
Pred continue
Pred continue
Pred continue
Pred continue
Pred continue
Koniec?
1. Iteracia
Koniec?
